### PR TITLE
[MANOPD-1] Update default version kubernetes

### PR DESCRIPTION
**Pull request by dmyar21**
### Description
* You need to update the default version of kubernetes in default.yaml

Fixes # (issue)
[MANOPD-86694]

### Solution
* Updated the default version of kubernetes to 1.26.3

### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [ ] There is no merge conflicts

#### Unit tests
Indicate new or changed unit tests and what they do, if any.

### CODE REVIEW

File: kubemarine/resources/configurations/defaults.yaml [hunk 1]
```diff
@@ -717,7 +717,7 @@ rbac:
           annotations: {}
         type: kubernetes.io/service-account-token
 
-  admission: psp
+  admission: pss
```
Comment (ilia1243) [inline kubemarine/resources/configurations/defaults.yaml:720]: This is incompatible change as Kubemarine suddenly starts to install different admission implementation on old clusters. I think that default admission implementation should be calculated dynamically based on kubernetes version if no explicit implementation is specified. @alexarefev

Comment (alexarefev) [reply kubemarine/resources/configurations/defaults.yaml:720]: Definitely we need an improvement here

File: kubemarine/resources/configurations/defaults.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ services:
     # for patches functionality we need v1beta3 which is available since k8s v1.22 only
     apiVersion: '{% if "v1.21" in services["kubeadm"]["kubernetesVersion"] %}kubeadm.k8s.io/v1beta2{% else %}kubeadm.k8s.io/v1beta3{% endif %}'
     kind: ClusterConfiguration
-    kubernetesVersion: v1.24.11
+    kubernetesVersion: v1.26.3
```
Comment (ilia1243) [inline kubemarine/resources/configurations/defaults.yaml:45]: Please also update schema/services/kubeadm.json. Also, I would like to remind that every update of defaults changes Kubemarine behaviour which can (highly unlikely) break some usages. https://github.com/Netcracker/KubeMarine/pull/345#discussion_r1100194109

**Review by ilia1243 — APPROVED** [review]

File: kubemarine/resources/configurations/defaults.yaml (hunk 2 above)
Comment (dmyar21) [reply kubemarine/resources/configurations/defaults.yaml:45]: update json schema

**Review by koryaga — APPROVED** [review]

**Pull request merged**